cms/templates/import.html

```diff
@@ -113,11 +165,23 @@ <h3 class="title">${_("Updating Course")}</h3>
 
               <div class="status-detail">
                 <h3 class="title">${_("Success")}</h3>
-                <p class="copy">${_("Your imported content has now been integrated into this course")}</p>
+                <p class="copy">
```

Comment:
@marcotuts I've actually already pushed a fix for the problem, but it's different than you mention. The Javascript now swaps out the icon with a checkmarked box when the import has completed, leaving it as a gray box until this happens. See `cms/static/js/views/import.js`